source/adapters/level_zero/queue.cpp

```diff
@@ -1983,8 +1987,10 @@ ur_command_list_ptr_t &ur_queue_handle_t_::ur_queue_group_t::getImmCmdList() {
     Priority = "High";
   }
 
-  // Evaluate performance of explicit usage for "0" index.
-  if (QueueIndex != 0) {
+  if (Queue->Device->useDriverInOrderLists() && Queue->isInOrderQueue()) {
+    ZeCommandQueueDesc.flags = ZE_COMMAND_QUEUE_FLAG_IN_ORDER;
```

Comment:
just out of curiosity: is the flag `ZE_COMMAND_QUEUE_FLAG_IN_ORDER` mutually exclusive with `ZE_COMMAND_QUEUE_FLAG_EXPLICIT_ONLY`?

Reply:
it is not mutually exclusive, ZE_COMMAND_QUEUE_FLAG_EXPLICIT_ONLY doesn't do anything in today's driver